shared-libs/user-management/src/users.js

```diff
@@ -452,14 +459,31 @@ const getSettingsUpdates = (username, data) => {
   return settings;
 };
 
-const getUserUpdates = (username, data) => {
+const isTargetAdminUser = async (username) => {
+  const admins = await couchSettings.getCouchConfig('admins');
+  return admins && !!admins[username];
+};
+
+const requirePasswordChange = async (username, fullAccess) => {
+  if (!fullAccess) {
+    return false;
+  }
+
+  return !(await isTargetAdminUser(username));
```

Comment:
You're correct, previous discussions with @garethbowen was to require password reset every time an admin changes a user's password for security purposes like you mention.

Reply:
Hmm, that does not seem to be the direction of the initial conversation out on the issue thread.  I added [a comment](https://github.com/medic/cht-core/issues/9547#issuecomment-2541646693) to the main issue to give this question a bit more visibility.  I am just a bit extra wary here since this feels a bit uncharacteristic compared to how we usually roll out CHT changes...